src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,17 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+        <UserInfo user={post.user} />
+      </div>
+
+      <p className="PostInfo__body">{post.body}</p>
+
+      <CommentList comments={post.comment} />
```

Comment:
There is a typo in the prop passed to `CommentList`. It should be `post.comments` instead of `post.comment` to correctly pass the comments array associated with the post.